src/app/globals.css

```
@@ -418,6 +418,233 @@
+  --mitti-forest-deep: #0a1510;
+  --mitti-lime: #e5ffb2;
+  --mitti-lime-soft: rgba(229, 255, 178, 0.36);
+  --mitti-blue: #0000ee;
+  --mitti-orange: #ff561c;
+  --mitti-sky: #a6c7de;
+  --mitti-sage: #6f8f5e;
+}
+
+/* Display font utility — Archivo at heavy weight, wide tracking */
+.font-display {
+  font-family: var(--font-display), 'Archivo', 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.1)

[error] Expected no quotes around "Archivo" (font-family-name-quotes)

(font-family-name-quotes)